### PR TITLE
IS-1005: Midnight option append on ICONLogger

### PR DESCRIPTION
**Pull request by cow-hs**
* append MIDNIGHT option on ICON Logger

### CODE REVIEW

**Comment by cow-hs** [comment]
find bug!
* weekly option is not working
we need to add more info ref https://docs.python.org/ko/3/library/logging.handlers.html into option params

**Comment by cow-hs** [comment]
> find bug!
> 
> * weekly option is not working
>   we need to add more info ref https://docs.python.org/ko/3/library/logging.handlers.html into option params

fixed

**Review by yakkle — APPROVED** [review]

File: iconcommons/logger/_logger/icon_logger_util.py [hunk 1]
```diff
@@ -64,24 +66,37 @@ def from_dict(cls, src_config: dict):
...
-            return 'W'
         elif value == 'hourly':
             return 'H'
         elif value == 'minutely':
             return 'M'
         elif value == 'secondly':
             return 'S'
+        elif value == 'midnight':
+            return 'MIDNIGHT'
+        elif "weekly" in value:
```
Comment (goldworm-icon) [inline iconcommons/logger/_logger/icon_logger_util.py]: How about checking for 'weekly' value more strictly using regular expression?
weekly[0-6]{1}

Comment (cow-hs) [reply iconcommons/logger/_logger/icon_logger_util.py]: fixed

**Review by goldworm-icon — APPROVED** [review]

**Pull request merged**